ctapipe/reco/energy_regressor.py

```diff
@@ -0,0 +1,383 @@
+import numpy as np
+
+from astropy import units as u
+
+from sklearn.ensemble import RandomForestRegressor
+
+
+def pick_any(my_dict):
```

Comment:
As from 3.6 on, python dictionaries are sorted (and this will most probably become a feature rather than an implementation detail) this will always return the first element, not an **arbitrary** element.

Also, this function is a oneliner:  `return next(iter(d.values()))` or if you want a random one: `random.choice(list(d.values()))`